pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -90,14 +90,15 @@ def __verify_train_loop_configuration(self, model):
         if (has_overriden_optimization_functions) and going_to_accumulate_grad_batches and automatic_optimization:
             raise MisconfigurationException(
                 'When overriding `LightningModule` optimizer_step or optimizer_zero_grad with '
-                '`Trainer(automatic_optimization=True, ...)`, `accumulate_grad_batches` should to be 1.'
+                '`Trainer(...)`, `accumulate_grad_batches` should to be 1.'
```

Comment:
```suggestion
                ' `Trainer(...)`, `accumulate_grad_batches` should to be 1.'
```
What does Trainer(...) mean here?